src/app/Services/data.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import data from '../../assets/messages';
```

Comment:
should use absolute path for these files: `'./src/assets/messages.ts'`

Reply:
import from the data.service.ts file, so the path must be from this file, if you use the path ./src/assets/messages.ts, you will get an error because the file is not found.

Reply:
try to use absolute path I mean. don’t use exactly what I wrote, haven’t test yet.